frameworks/hdfs/universe/config.json

```diff
@@ -63,55 +63,59 @@
           "enum": ["OFF", "FATAL", "ERROR", "WARN", "INFO", "DEBUG", "TRACE", "ALL"],
           "default": "INFO"
         },
-        "tls": {
+        "security": {
+          "description": "HDFS security settings.",
           "type": "object",
-          "description": "TLS configuration properties.",
-          "properties": {
-            "enabled": {
-              "description": "Enable TLS support (requires Strict mode).",
-              "type": "boolean",
-              "default": false
+          "tls": {
```

Comment:
I minor nit. Could we move this to after the `kerberos` section like the other frameworks in #1871?

Alternatively, we can address the TLS move as a single PR there?

Reply:
Done